src/types.js

```diff
@@ -244,8 +348,55 @@ class FP_TimeBase extends FP_Type {
     }
     return this.dateObj;
   }
+
+
+  /**
+   *  Creates a date object for the given timezone.  The returned date object
+   *  will have the specified date and time in the specified timezone.
+   * @param year...ms Just as in the Date constructor.
+   * @param timezoneOffset (optiona) a string in the format (+-)HH:mm or Z, representing the
+   *  timezone offset.  If not provided, the local timzone will be assumed (as the
+   *  Date constructor does).
+   */
+  _createDate(year, month, day, hour, minutes, seconds, ms, timezoneOffset) {
+    var d = new Date(year, month, day, hour, minutes, seconds, ms);
+    if (timezoneOffset) {
```

Comment:
Can the input date/time component params be null/undefined? if not, would it work to simply construct a string here and give it to new Date()?

Reply:
It looks like chrome supports null parameters; I don't know if that is standard.  How would that help?  With regard to "new Date()", the [Mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) has this warning: 

```
Parsing of date strings with the Date constructor (and Date.parse(), which works the same way) is strongly discouraged due to browser differences and inconsistencies. Support for RFC 2822 format strings is by convention only. Support for ISO 8601 formats differs in that date-only strings (e.g. "1970-01-01") are treated as UTC, not local.
```